lib/style-observer.js

```diff
@@ -454,12 +459,17 @@ var StyleObserver = (function() { // jshint ignore:line
      * Creates a new pseudo-class and registers it in Sizzle
      */
     var extendSizzle = function(Sizzle) {
-        Sizzle.selectors.pseudos["properties"] = Sizzle.selectors.pseudos["-abp-properties"] = Sizzle.selectors.createPseudo(function(propertyFilter) {
+        Sizzle.selectors.setFilters["properties"] = Sizzle.selectors.pseudos["-abp-properties"] = Sizzle.selectors.createPseudo(function(propertyFilter) {
```

Comment:
If you're using `createPseudo` function it does not matter if `setFilters` or `pseudos` is used (Sizzle checks for `fn[expando]` so I suggest rolling it back to `pseudos`).

Reply:
2. [ ] All the setFilters changes should be rolled back